app/jobs/application_job.rb

```diff
@@ -3,7 +3,11 @@
 class ApplicationJob < ActiveJob::Base
 
   rescue_from(StandardError) do |e|
-    NewRelic::Agent.notice_error(e)
+    begin
+      NewRelic::Agent.notice_error(e)
+    ensure
+      raise e
```

Comment:
This seems to be the main thing, I haven't tested this, but my understanding and from looking at the docs is that raised errors should lead to retries b/c the message won't be deleted, and so should reappear when it is no longer hidden in SQS